Functions/Assertions/Should.ps1

```diff
@@ -83,11 +84,13 @@ function Should {
             $testFailed = Get-TestResult $parsedArgs $value
 
             if ($testFailed) {
+                $ShouldExceptionLineText = $MyInvocation.Line.TrimEnd("`n")
```

Comment:
I thought about it and I think preserving original indent is a good thing.
One of the problems in current implementation is multi-line statement
```powershell
It "should foo" {
   Invoke-Foo -ScriptBlock {
   ...
   } | Should be 'foo'
}
```
One possible improvement is to detect this the whole pipeline statement and show multi-line. Then preserving indents will be mandatory.